server/src/test.json

```diff
@@ -4,919 +4,839 @@
         "likes": 1,
         "text": "Jon is testing questions",
         "hidden": true,
-        "answered": false,
         "createTimeUnix": 1667437961
     },
     {
         "id": 1,
         "likes": 92,
         "text": "How do you manage your time for work, open source, personal projects, and life?",
         "hidden": false,
-        "answered": true,
+        "answered": 1670131269,
```

Comment:
I'd like to keep this file as-is, since it's a pure export file from live-ask.com.

Reply:
Reverted.